chrononaut/flask_versioning.py

```diff
@@ -2,46 +2,71 @@
 """
 from flask import g
 from flask.globals import _app_ctx_stack
+from datetime import datetime
+from dateutil.tz import tzutc
+import pytz
```

Comment:
`pytz` and `dateutil.tz` do the same thing but treat timezone objects differently:
https://blog.ganssle.io/articles/2018/03/pytz-fastest-footgun.html

Use only one. I would suggest to use `dateutil.tz`. It is saner than `pytz`.